fulp_modules/main_features/bloodsuckers/code/powers/_powers.dm

```diff
@@ -332,21 +345,22 @@
 	// Valid? (return true means DON'T cancel power!)
 	if(!CheckCanPayCost(TRUE) || !CheckCanUse(TRUE) || !CheckCanTarget(A, TRUE))
 		return TRUE
+	power_in_use = TRUE	 // Lock us into this ability until it successfully fires off. Otherwise, we pay the blood even if we fail.
+	FireTargetedPower(A) // We use this instead of ActivatePower(), which has no input
 	// Skip this part so we can return TRUE right away.
 	if(power_activates_immediately)
 		PowerActivatedSuccessfully() // Mesmerize pays only after success.
-	power_in_use = TRUE	 // Lock us into this ability until it successfully fires off. Otherwise, we pay the blood even if we fail.
-	FireTargetedPower(A) // We use this instead of ActivatePower(), which has no input
 	power_in_use = FALSE
 	return TRUE
 
 /// Like ActivatePower, but specific to Targeted (and takes an atom input). We don't use ActivatePower for targeted.
 /datum/action/bloodsucker/targeted/proc/FireTargetedPower(atom/A)
+	var/mob/living/target = A
```

Comment:
just make the first argument a living target

Reply:
this one is acutally my mistake because targeted powers can be used on more than just mobs